src/interactive/azext_interactive/tests/latest/cache/help_dump_test.json

```diff
@@ -814,13 +814,6 @@
                 "required": "",
                 "help": "enable accelerated networking"
             },
-            "--public-ip-per-vm": {
-                "name": [
-                    "--public-ip-per-vm"
-                ],
-                "required": "",
-                "help": "Each VM instance will have a public ip. For security, you can use '--nsg' to apply appropriate rules"
-            },
```

Comment:
Why delete them?

Reply:
This one is not defined correctly and is replaced by --enable-node-public-ip in this PR.